Libraries/LibWeb/Streams/ReadableByteStreamController.h

```diff
@@ -22,9 +23,15 @@ enum class ReaderType {
 
 // https://streams.spec.whatwg.org/#pull-into-descriptor
 struct PullIntoDescriptor : public GC::Cell {
+private:
     GC_CELL(PullIntoDescriptor, GC::Cell);
     GC_DECLARE_ALLOCATOR(PullIntoDescriptor);
 
+    IntrusiveListNode<PullIntoDescriptor> m_list_node;
+
+public:
+    using List = IntrusiveList<&PullIntoDescriptor::m_list_node>;
```

Comment:
I'm not 100% convinced this is better since there are some cases that we do need to have a List<GC::Root>, and I don't know if there is a nice way of doing that and if it's worth the tradeoff.

Reply:
I ended up undoing this but have kept the other change I noticed as part of this